c-impl/PQCrypto-SIDH/src/poik.c

```diff
@@ -1234,7 +1254,7 @@ static void render_progress(size_t idx)
     }
 
     static size_t last = -1;
-    unsigned const done = 64 * idx / POIK_REPS;
+    unsigned const done = 63 * idx / POIK_REPS;
```

Comment:
63?

Reply:
64 was off by one, causing undefined behaviour in theory and an occasional visual glitch in the progress bar in practice... 😬

Reply:
oof!